examples/OrderExample.tsx

```
@@ -0,0 +1,202 @@
+  tpsl: Tpsl;
+}
+
+interface OrderType {
+  limit?: LimitOrderType;
+  trigger?: TriggerOrderType;
+}
+
+class Cloid {
+  private _rawCloid: string;
+
+  constructor(rawCloid: string) {
+    this._rawCloid = rawCloid;
+    this._validate();
+  }
+
+  private _validate(): void {
+    if (!this._rawCloid.startsWith("0x")) {
+      throw new Error("cloid is not a hex string");
+    }
+    if (this._rawCloid.slice(2).length !== 32) {
+      throw new Error("cloid is not 16 bytes");
+    }
+  }
+
+  static fromInt(cloid: number): Cloid {
+    return new Cloid(`0x${cloid.toString(16).padStart(32, "0")}`);
+  }
+
+  static fromStr(cloid: string): Cloid {
+    return new Cloid(cloid);
+  }
+
+  toRaw(): string {
+    return this._rawCloid;
+  }
+}
+
+interface OrderRequest {
+  coin: string;
+  is_buy: boolean;
+  sz: number;
+  limit_px: number;
+  order_type: OrderType;
+  reduce_only: boolean;
+  cloid?: Cloid | null;
+}
+
+function floatToWire(x: number): string {
+  const rounded = x.toFixed(8);
+  if (Math.abs(parseFloat(rounded) - x) >= 1e-12) {
+    throw new Error("floatToWire causes rounding");
+  }
+  if (rounded === "-0") {
+    return "0";
+  }
+  return new Decimal(rounded).toString();
+}
+
+function orderTypeToWire(orderType: OrderType): OrderTypeWire {
+  if ("limit" in orderType) {
+    return { limit: orderType.limit };
+  } else if ("trigger" in orderType && orderType.trigger) {
+    return {
+      trigger: {
+        isMarket: orderType.trigger.isMarket,
+        triggerPx: floatToWire(orderType.trigger.triggerPx),
+        tpsl: orderType.trigger.tpsl,
+      },
+    };
+  }
+  throw new Error("Invalid order type");
+}
+
+const orderRequest: OrderRequest = {
+  coin: "BTC",
+  is_buy: true,
+  sz: 0.001,
+  limit_px: 90000,
+  reduce_only: false,
+  order_type: {
+    limit: { tif: "Gtc" }, // Gtc: Good till Cancel
+  },
+};
+
+const assetId = 0;
+const activePool = null;
+const wallet = privateKeyToAccount("0x-your-secret-key");
```

What do you think of adding a description at the top that mentions that you need to set your secret key and pulling this into a separate constant. Can also throw an error saying to set it instead of relying on the default bad key error.